plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryConfig.java

```diff
@@ -62,6 +63,9 @@
     private int rpcMaxChannelCount = 1;
     private int minRpcPerChannel;
     private int maxRpcPerChannel = Integer.MAX_VALUE;
+    private int retries;
```

Comment:
Should we use some retries by default to make it more stable? Maybe 3?

Reply:
Default is 0